### PR TITLE
Configure Crashlytics script to run

### PR DESCRIPTION
**Pull request by daniel-chuang**
## Overview

Configured the Firebase Crashlytics script to run.

## Changes Made

Modified the Xcode build configurations according to the AppDev notion page on Crashlytics. Used a slightly different script to account for the fact that we are using Cocoapods rather than Swift Package Manager.

## Test Coverage

The app compiles and runs. However, we still need to test whether the logging works with our Pod lead to see the dashboard!

### CODE REVIEW

**Review by vinnie4k — COMMENTED** [review]
Great job on your first PR! I will approve once we have confirmed that crashes show up on Firebase. Also, we typically don't do draft PRs but it's fine if you do.

File: TCAT.xcodeproj/project.pbxproj
Comment (belle-hu) [inline TCAT.xcodeproj/project.pbxproj:1]: Looks good! I think the new code matches the Firebase Crashlytics instructions perfectly, and the style matches all the other lines in the file.

**Comment by cassidyxu** [comment]
Looks good!

**Review by belle-hu — APPROVED** [review]

**Review by JaysonHahn — APPROVED** [review]
Looks good to me!!

**Pull request merged**